tests/test_bundle_workflow.py

```diff
@@ -164,6 +167,29 @@ def test_non_config_wrong_log_cases(self, meta_file, logging_file, expected_erro
         with self.assertRaisesRegex(FileNotFoundError, expected_error):
             NonConfigWorkflow(self.filename, self.data_dir, meta_file, logging_file)
 
+    def test_pythonic_workflow(self):
+        device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+        config_file = {"roi_size": (64, 64, 32)}
+        meta_file = os.path.join(os.path.dirname(__file__), "testing_data", "metadata.json")
+        workflow = PythonicWorkflowImpl(workflow_type="infer", config_file=config_file, meta_file=meta_file)
```

Comment:
Can we use the `create_workflow` API to create a pythonic workflow? If so, could you please provide an example here?

Thanks,  
Bin

Reply:
Thanks, I added related test, please help review again.